services/inventory/grpc/services_server.go

```diff
@@ -39,7 +39,10 @@ func NewServicesServer(s *inventory.ServicesService) inventorypb.ServicesServer
 
 // ListServices returns a list of all Services.
 func (s *servicesServer) ListServices(ctx context.Context, req *inventorypb.ListServicesRequest) (*inventorypb.ListServicesResponse, error) {
-	services, err := s.s.List(ctx)
+	filters := inventory.ServiceFilters{
+		NodeID: req.GetNodeId(),
```

Comment:
In general, prefer fields to methods. `reg.NodeId`. Methods are useful _sometimes_ (for chains where `nil` is possible), but also send a signal that _something is complected there_, and that place is a simple one.

Reply:
we can't use field here because in proto file we used `oneof`

Reply:
ok